server/tokenRegistryProxy.js

```diff
@@ -0,0 +1,46 @@
+require('isomorphic-fetch')
+const Sentry = require('@sentry/node')
+const {backendConfig} = require('./helpers/loadConfig')
+const mockData = require('./_tokenRegistryMock')
+const cachedFetch = require('./helpers/cachedFetch')
+
+const CACHE_TIMEOUT = 24 * 60 * 60 * 1000 // 1 day
```

Comment:
Do we know the actual rate limits of the API? I tried to check https://github.com/cardano-foundation/cardano-token-registry/blob/master/API_Terms_of_Use.md but didn't find any clear information on this

Reply:
I didn't find any information on rate limits as well.

Reply:
I raised a question in the repo: https://github.com/cardano-foundation/cardano-token-registry/issues/291 - feel free to follow it